product_docs/docs/pgd/5/reference/commit-scopes.mdx

```diff
@@ -46,57 +50,57 @@ where `node_group` is the name of a PGD data node group.
 
 Example: `ANY 2 (left_dc)`
 
-A transaction under this commit scope group will be considered commited if any two nodes in the left_dc group confirm they have processed the transaction.
+A transaction under this commit scope group is considered committed if any two nodes in the `left_dc` group confirm they processed the transaction.
```

Comment:
I feel like this is flattening out the temporal nature of the rule... "will be" is used because the rule in the scope is created before any transaction is processed. These rule elements are used in advance of transactions being processed and formulated by the reader.

Reply:
What if you changed "if" to "after"? Then you'd have the temporal nature, probably with more accuracy?

Reply:
So "A transaction under this commit scope group will be considered committed after any two nodes in the...."

Sounds good to me.

Reply:
```suggestion
A transaction under this commit scope group will be considered committed after any two nodes in the `left_dc` group confirm they processed the transaction.
```

Reply:
✔️